### PR TITLE
adds Kinyarwanda language support

### PR DESCRIPTION
**Pull request by Emile-Nsengimana**
#### Is your pull request related to a problem? Please describe
This pull request adds Kinyarwanda (rw) language support as per the issue [#57](https://github.com/fbdevelopercircles/open-source-edu-bot/issues/57)

#### Additional context
screen-shoot

#### Checklist
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] I have added the label **enhancement** to my pull request.

### CODE REVIEW

**Comment by Emile-Nsengimana** [comment]
Thank you @elinguiuriel

File: src/services/profile.py [hunk 1]
```diff
@@ -45,6 +45,12 @@
             "text": u'🙋🏽 ආයුබෝවන් {{user_first_name}}! විවෘත මූලාශ්‍ර 🔓 වලට'
             ' අදාල Facebook DevC වලින් ලබාදෙන සම්පත් වලට ප්‍රවේශ වෙන්න,'
             ' Get Started බටන් එක ක්ලික් කරන්න.'
+        },
+        {
+            "locale": "rw_RW",
+            "text": u'🙋🏽 Muraho {{user_first_name}}! Kanda kuri Buto Tangira '
+            'hepfo kugirango ugere kuri Facebook DevC ibikoresho byegeranyijwe bijyanye'
```
Comment (elinguiuriel) [inline src/services/profile.py:52]: @Emile-Nsengimana before we merge you PR please correct this simple [issue](https://github.com/fbdevelopercircles/open-source-edu-bot/pull/99/checks): Line 52 should be broken, it is too long.

```bash
============================= test session starts ==============================
platform linux -- Python 3.7.8, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /app, inifile: setup.cfg, testpaths: tests
collected 6 items

tests/test_factory.py .                                                  [ 16%]
tests/test_services_messenger.py ....                                    [ 83%]
tests/test_services_profile.py .                                         [100%]

============================== 6 passed in 0.35s ===============================
./services/profile.py:52:80: E501 line too long (88 > 79 characters)
##[error]Process completed with exit code 1.
```

Comment (Emile-Nsengimana) [reply src/services/profile.py]: done @elinguiuriel

File: src/locales/rw/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+msgid "Friend"
+msgstr "Nshuti"
+
+#: services/messenger.py:75
+#, python-format
+msgid ""
+"🙏🏼 Hi %(first_name)s, so you’ve decided to make your first steps in Open "
+"Source. That’s great."
+msgstr ""
+"🙏🏼 Muraho% (first_name), nuko rero wahisemo gutera intambwe zawe za mbere"
```
Comment (elinguiuriel) [inline src/locales/rw/LC_MESSAGES/messages.po:30]: @Emile-Nsengimana bad translation makes sure to write `Muraho %(first_name)s, nuko....`

File: src/locales/rw/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+msgstr "✔️ Ego"
+
+#: services/messenger.py:87 services/messenger.py:203 services/messenger.py:222
+msgid "❌ Not yet"
+msgstr "❌ Oya"
+
+#: services/messenger.py:90
+#, python-format
+msgid "So, tell me %(first_name)s do you know what Open Source is? 👇🏼"
+msgstr "Noneho, mbwira% (first_name) uzi Open source icyo aricyo? 👇🏼"
```
Comment (elinguiuriel) [inline src/locales/rw/LC_MESSAGES/messages.po:44]: @Emile-Nsengimana bad translation `.. %(first_name)s ...`

File: src/locales/rw/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -0,0 +1,627 @@
...
+#: services/messenger.py:155
+#, python-format
+msgid ""
+"I didn't get you %(first_name)s!\n"
+"You said : %(msg)s\n"
+"\n"
+" This is the main menu, select what you need below 👇🏼"
+msgstr ""
+"Sinumvise %(first_name)s!\n"
+"Wavuze:% (msg)s\n"
```
Comment (elinguiuriel) [inline src/locales/rw/LC_MESSAGES/messages.po:87]: @Emile-Nsengimana error `... %(msg)s\n`

Comment (Emile-Nsengimana) [reply src/locales/rw/LC_MESSAGES/messages.po:87]: I saw you have already fix the bug @elinguiuriel 🙏

**Comment by elinguiuriel** [comment]
It seems to work as expected now, after #104 

image

image

**Pull request merged**